source/jormungandr/jormungandr/scenarios/experimental.py

```diff
@@ -243,9 +267,12 @@ def call_kraken(self, request_type, request, instance, krakens_call):
         """
         logger = logging.getLogger(__name__)
         logger.debug('datetime: %s', request['datetime'])
-        crow_fly_stop_points = set()
+        #We use a dict with a key/value of stop_point.uri and is_zonal
+        #a crow_fly section is added only if stop_point is not zonal
```

Comment:
So `crow_fly_stop_points['toto']` correspond to 'toto is zonal odt? the variable name is quite illisible.

I don't have any good name to propose.